src/models/PendingTalkClaimMapper.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/**
+ * PendingTalkClaimMapper
+ *
+ * @uses ApiModel
+ * @package API
+ */
+class PendingTalkClaimMapper extends ApiMapper
+{
+
+    /**
+     * Default mapping for column names to API field names
+     *
+     * @return array with keys as API fields and values as db columns
+     */
+    public function getDefaultFields()
+    {
+        $fields = array(
```

Comment:
No short array syntax? Looks like `composer.lock` requires min PHP of `>=5.4.0` (though, oddly, it's not mentioned in `composer.json`), so short syntax is available.